src/FormBuilder/resources/js/form-builder/src/common/popover.js

```diff
@@ -0,0 +1,23 @@
+export default function popover() {
+    window.addEventListener('wheel', (event) => {
+        const popover = document.querySelector('.components-popover.block-editor-block-popover');
+
+        if (!!popover) {
+            const {top} = popover.getBoundingClientRect();
+
+            if (top < 150) {
+                const topStyle = Math.sign(event.deltaY) + 1;
+                popover.style.top = topStyle + 'rem';
+            } else {
+                popover.style.top = '0rem';
+            }
+        }
+    });
+
+    window.addEventListener('click', (event) => {
+        const popover = document.querySelector('.components-popover.block-editor-block-popover');
+        if (!!popover) {
+            popover.style.top = '0rem';
+        }
+    });
+}
```

Comment:
@glaubersilva haha this looks fun! 
Thanks for the video explaining why you were working on this.  Looking at this and what the post editor is doing, i'm pretty sure we're trying to compensate for css issue on the form builder.  The reason is the block popover is calculating it's position based on it's relation to the window/document and block - so the reason it would be off is up the chain in our document somewhere.

Doing a quick look I was able to see our `.interface-interface-skeleton` is not actually full height and has a position top - while our `interface-interface-skeleton__header` header container might not need to position fixed.

I guarantee fixing the document css / height issues will resolve the popover problem.

Reply:
@jonwaldstein Ohh thanks! I spent more time than necessary on this one. 🤦‍♂️

Now it's resolved in 7a679bf61554d286b3c1bc81eeb2540b2e921e67 and 791bb1bad4cb9cc69242ddc40602b1e4bb921a09

Reply:
Haha awesome, thanks!